.azure/dev-template.yml

```diff
@@ -54,7 +54,7 @@ jobs:
 
     - script:
         make testpy-forked
-      displayName: 'pytest --forked'
+      displayName: 'pytest -n 2'
```

Comment:
Maybe just `pytest distributed` or `pytest parallelized` or similar. And the same name for the Makefile target.